test/e2e/network_segmentation.go

```diff
@@ -2503,3 +2558,25 @@ func unmarshalPodAnnotationAllNetworks(annotations map[string]string) (map[strin
 	}
 	return podNetworks, nil
 }
+
+// takes ipv4 and ipv6 cidrs and returns the correct type for the cluster under test
+// this function is useful when cidr has both cluster and host subnets included.
+// example: "192.168.100.0/25/26", "2014:100:200::/63/64"
+func correctCIDRFamily(cs clientset.Interface, ipv4CIDR, ipv6CIDR string) string {
+	return strings.Join(selectCIDRs(cs, ipv4CIDR, ipv6CIDR), ",")
+}
+
+// takes ipv4 and ipv6 cidrs and returns the correct type for the cluster under test
+func selectCIDRs(cs clientset.Interface, ipv4CIDR, ipv6CIDR string) []string {
```

Comment:
here you probably want to use the existing `filterCIDRs`

Reply:
@ricky-rav right, but `filterCIDRs` don't work when CIDR having two two slashes (/), so using these new functions to support it.